l10n_it_fatturapa_out/views/account_view.xml

```diff
@@ -1,25 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <record id="view_invoice_fatturapa_out_tree" model="ir.ui.view">
+        <field name="name">view.invoice.fatturapa.out.pec.tree</field>
```

Comment:
Qui è rimasto un `.pec.`.
Dato che hai portato qui la tree, valuterei eventualmente di unifomare l'id e il nome con la vista form.

Reply:
> Dato che hai portato qui la tree, valuterei eventualmente di unifomare l'id e il nome con la vista form.

Ad esempio, anche questo mio suggerimento era volto a migliorare la leggibilità, uniformando i nomi tra tree, form e search/filter, per semplificare il lavoro a eventuali nuovi contributori.

Bastava anche un no, motivando la risposta. :wink:

Reply:
> > Dato che hai portato qui la tree, valuterei eventualmente di unifomare l'id e il nome con la vista form.
> 
> Ad esempio, anche questo mio suggerimento era volto a migliorare la leggibilità, uniformando i nomi tra tree, form e search/filter, per semplificare il lavoro a eventuali nuovi contributori.
> 
> Bastava anche un no, motivando la risposta. 😉

A volte si perde più tempo a motivare la risposta che a modificarlo; in particolare questa PR è bloccante per un altro modulo.

Reply:
> > > Dato che hai portato qui la tree, valuterei eventualmente di unifomare l'id e il nome con la vista form.
> > 
> > 
> > Ad esempio, anche questo mio suggerimento era volto a migliorare la leggibilità, uniformando i nomi tra tree, form e search/filter, per semplificare il lavoro a eventuali nuovi contributori.
> > Bastava anche un no, motivando la risposta. wink
> 
> A volte si perde più tempo a motivare la risposta che a modificarlo;

Concordo, ma in questo caso non hai fatto nessuna delle due cose. 
Il tempo è prezioso per tutti, anche per chi fa le revisioni. :-)

Reply:
> > > > Dato che hai portato qui la tree, valuterei eventualmente di unifomare l'id e il nome con la vista form.
> > > 
> > > 
> > > Ad esempio, anche questo mio suggerimento era volto a migliorare la leggibilità, uniformando i nomi tra tree, form e search/filter, per semplificare il lavoro a eventuali nuovi contributori.
> > > Bastava anche un no, motivando la risposta. wink
> > 
> > 
> > A volte si perde più tempo a motivare la risposta che a modificarlo;
> 
> Concordo, ma in questo caso non hai fatto nessuna delle due cose.
> Il tempo è prezioso per tutti, anche per chi fa le revisioni. :-)

Come no?! Puoi verificare l'ultimo commit? 

https://github.com/OCA/l10n-italy/blob/54cd008ca488cf63129398a9d5839069e4c3b5cb/l10n_it_fatturapa_out/views/account_view.xml#L5

Reply:
Ah, no, aspetta. Non mi riferivo al fatto di togliere `.pec.` (che hai corretto) ma al suggerimento successivo.
`Dato che hai portato qui la tree, valuterei eventualmente di unifomare l'id e il nome con la vista form.`

Dopo lo spostamento delle due viste da fatturapa_pec a fatturapa_out:

_id_ `view_invoice_fatturapa_out_tree`
_name_ `view.invoice.fatturapa.out.tree`

_id_ `view_invoice_form_fatturapa`
_name_ `account.invoice.fatturapa`

_id_ `view_account_invoice_fatturapa_out_filter`
_name_ `view.account.invoice.fatturapa.out.filter`

Reply:
ah ok, vedi ultimo commit

Reply:
> ah ok, vedi ultimo commit

Ok, hai anche verificato e testato questa modifica in locale compreso l'aggiornamento?
Il "valuterei "che ti avevo suggerito presupponeva anche questo tipo di controlli (per questo anche un no motivato poteva andare bene :-) )

Tra l'altro visto che hai modificato id e nome della vista esistente di l10n_it_fatturapa_out (e non le due spostate), mi sa che ricadiamo su ciò che giustamente diceva Lorenzo [qui](https://github.com/OCA/l10n-italy/pull/2136#discussion_r585561348) e a cui non avevo pensato. Che ne pensi @eLBati ?

Reply:
non, ci capisco più una beata fava. mi, state rintronando con tutte questi commenti. 

Ditemi cosa diamine c'è da fare e lo faccio, altrimenti è una perdita di tempo; in caso contrario sentitevi liberi di modificarlo come meglio credete.

Se per ogni pr, anche banale ci va un mese, pare chiaro che ne passa la voglia di contribuire..

Reply:
> non, ci capisco più una beata fava. mi, state rintronando con tutte questi commenti.

Attenzione che tutti gli ultimi commenti sono nati da un equivoco, non avevi compreso una osservazione che avevo fatto (o forse non mi ero espresso bene).
La valutazione che ti avevo suggerito non era essenziale o bloccante, infatti avevo già approvato la PR 4 giorni fa dando per scontato che il tuo silenzio fosse un no.

![immagine](https://user-images.githubusercontent.com/227940/109775112-610d3880-7c01-11eb-95be-a6f71253d59b.png)

Detto questo, se hai fretta del merge elimina l'ultimo commit e procediamo.

Reply:
@primes2h infatti non ci stavo più a capire nulla; comunque ho eliminato l'ultimo commit. Poi nel caso ci si rimette mano e si sistema il resto.